consensus/state.go

```diff
@@ -946,14 +947,25 @@ func (cs *ConsensusState) createProposalBlock() (block *types.Block, blockParts
 		return
 	}
 
+	maxBytes := cs.state.ConsensusParams.BlockSize.MaxBytes
+	// bound evidence to 1/10th of the block
+	evidence := cs.evpool.PendingEvidence(maxBytes / 10)
```

Comment:
note: incoming block evidence is unbound

Reply:
We should state this ratio more explicitly

Reply:
Spec for how we construct a block?